src/ai/susi/server/api/aaa/UploadSettingsService.java

```diff
@@ -0,0 +1,97 @@
+/**
```

Comment:
It's a copyright header which all the servlets from aaa must have. I do not see where it deals with  code quality?